test-stories/timing.stories-disabled.js

```diff
@@ -19,7 +18,7 @@ const WaitFor = ({ seconds }) => {
 
   return (
     <div>
-      {Array.from(new Array(seconds - count)).map((x, index) => (
+      {Array.from(new Array(seconds - count)).map((_x, index) => (
```

Comment:
Do we want to fully elide these things?

```suggestion
      {Array.from(new Array(seconds - count)).map((_, index) => (
```

Reply:
Yeah I was on the fence with that one. We can go with `_`!